messages/deploy.metadata.validate.md

```diff
@@ -12,6 +12,8 @@ This command doesn't support source-tracking. When you quick deploy with the res
 
 To validate the deployment of multiple metadata components, either set multiple --metadata <name> flags or a single --metadata flag with multiple names separated by spaces. Enclose names that contain spaces in one set of double quotes. The same syntax applies to --manifest and --source-dir.
 
+Note: This command is not intended to be used on Sandboxes. By default, Sandboxes do not run tests during a deploy. To verify a deploy with tests on a Sandbox, use `sf project deploy start --dry-run --test-level RunLocalTests`.
```

Comment:
```suggestion
Note: Don't use this command on sandboxes.  By default, sandboxes don't run tests during a deploy.  If you want to validate a deployment with tests on a sandbox, use "sf project deploy start --dry-run --test-level RunLocalTests" instead.
```

Reply:
Same as above -- I edited the comment, but I still think more explanation is required.